modules/bene_media_instagram/bene_media_instagram.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update routines.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function bene_media_instagram_install() {
+  if (\Drupal::isConfigSyncing()) {
+    return;
+  }
+
+  bene_media_instagram_update_8200();
```

Comment:
I'm curious about this one. If you are installing the module, doesn't it already have the tags field coming from the install config yml files?

Reply:
@mariacha `bene_media_instagram` doesn't provide any default config, it all comes from `media_entity_instagram`.